latex/Specific Aims.tex

```diff
@@ -104,14 +104,14 @@
 Computational screening techniques can do better than random selection of compounds, but they lack the accuracy to guide molecular design or optimization. 
 A new generation of physical techniques, alchemical free energy calculations, are poised to fill this void by providing a quantitative, predictive tool that can be used in multiple stages of the drug discovery pipeline, including lead optimization to improve affinity and selectivity or the retention of potency as other physical properties are optimized. 
 
-Recent success of alchemical methods sparked considerable enthusiasm, but the domain of applicability of these techniques is currently highly limited; for broad application and routine use, they need further evaluation, refinement, and development. 
+Recent success of alchemical methods in predicting accurate affinities has sparked considerable enthusiasm, but the domain of applicability of these techniques is currently highly limited; broad application and routine use will require further evaluation, refinement, and development. 
```

Comment:
I like the edit in the first part of the sentence; less a fan of the second half (moving from active voice to passive, though it DOES make it more concise). I'll try to come up with an alternate phrasing for the second part after merging.